source/MaterialXCore/Util.cpp

```diff
@@ -91,6 +98,16 @@ StringVec splitString(const string& str, const string& sep)
     return split;
 }
 
+string mergeStringVec(const StringVec& stringVec, const string& sep)
```

Comment:
This is a good addition to the string library, though I would recommend the name `joinStrings`, in the spirit of the `String.join` method in Python.